Progressus-io/postnl-for-woocommerce
# Fix/bulk merge pdf

### Description
Fix the merge PDF issue when doing bulk operation.

### Steps to Test
1. Create a couple of Order with PostNL as the shipping method.
2. Use bulk operation in order list page to process the PostNL.
3. Notice that the merged pdf is now more efficient in page.
